js/sponsor.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(()=>{
+
+    $.get('./data/sponsors.json',(response) => {
+        constructSponsorBlock(response);
+    });
+});
+
+constructSponsorBlock = function(response){
+        
+        var text = "";
+        $.each(response,(key,value) => {
+            if(!(key==="border")){
```

Comment:
I don't feel we should be using a additional  conditional here. Since border image is fixed just hard code it.

Reply:
Okay will rewrite it some other way